warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -66,9 +66,25 @@ def open_browser(self, browser_name='firefox', webdriver_remote_url=False,
         browser_name = browser_name
         browser = self._make_browser(browser_name, desired_capabilities,
                                      profile_dir, webdriver_remote_url,
-                                     binary=binary, gecko_path=gecko_path)
+                                    binary=binary, gecko_path=gecko_path)
+        print_info("The Selenium Webdriver version is '{0}'".format(webdriver.__version__))
+        if browser:
+            browser_version = self.get_browser_details(browser)
+            print_info("The Browser '{0}' version is '{1}'".format(browser_name, browser_version))
         return browser
 
+    def get_browser_details(self, browser):
+        """ Return the Browser details
+
+            Arguments :
+                       browser = The browser instance
+
+            Return :
+                     browser_version = The browser instances' version.
+        """
+        browser_version = browser.capabilities['version']
+        return browser_version
+
```

Comment:
It would be better to return a dictionary with version as key. New fields can be added on the fly if needed.